src/web/services/participant.ts

```diff
@@ -99,6 +99,18 @@ export async function GetApprovedParticipants() {
   }
 }
 
+export async function GetParticipantApiKeys(participantId?: number) {
+  try {
+    const result = await axios.get<ApiKeyDTO[]>(
+      `/participants/${participantId ?? 'current'}/apiKeys`
+    );
+
+    return result.data;
+  } catch (e: unknown) {
+    throw backendError(e, 'Could not get the apiKeys');
```

Comment:
We should assume that this message will end up getting shown on the UI. In that case, let's change "apiKeys" -> "API keys"